src/theme/components/tooltip.ts

```diff
@@ -0,0 +1,14 @@
+import { cssVar } from "@chakra-ui/theme-tools";
+
+const $arrowBg = cssVar("popper-arrow-bg");
+
+const Tooltip = {
+  baseStyle: {
+    [$arrowBg.variable]: "colors.ui.gray.xx-dark",
```

Comment:
This is interesting. How does it work?

Reply:
It overwrites the existing CSS variable defined in the [Chakra Tooltip theme](https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/tooltip.ts). The variable is used to set the background of the Chakra component. This needs to be done since the popper arrow is not part of the tooltip body. 

I'm going to update this to use "tooltip-bg" instead of "popper-arrow-bg" since it can be used to set the bg once.

You can read more about it here: https://chakra-ui.com/docs/styled-system/css-variables

Reply:
Makes sense. This Chakra component doesn't break the styles apart like other components, so this is new to me.